codemagic.yaml

```diff
@@ -0,0 +1,222 @@
+# Has a separate flows for develop branch (automatic builds),
+# Merge Request (automatic builds) and master branch Release builds (manual builds).
```

Comment:
The "and master branch Release builds (manual builds)." I think you can release as e do not release any artifacts from here